src/pages/Rewards/index.tsx

```diff
@@ -58,13 +59,15 @@ const ButtonRow = styled(RowFixed)`
     margin-bottom: 8px;
   `};
 `
+type CurrencySearchParams = {
+  currencyIdA: string
+  currencyIdB: string
+}
```

Comment:
Same here

Reply:
The problem is where is the single location. Since its across different pages, I intentionally kept it separate for maintainability. Since its not a shared type, I felt repeating is fine. If you all feel it should be DRY i can create a global type file. But personally I like to keep types close to the component.